finance-api/user-guides/fetch-account-transactions.md

```diff
@@ -0,0 +1,150 @@
+Fetch account transactions
+===
+Using the Finance API, you can fetch all transactions or transactions of certain types from a merchant's Zettle liquid account during a specific period.
+
+> **Note**: A merchant's Zettle liquid account contains all confirmed transactions that are already paid out or to be paid out to the merchant's bank account. If you want to check preliminary transactions that are being confirmed with buyers' banks, you can fetch them from a merchant's Zettle preliminary account.
+
+<!-- Is there any limit for how old transactions can be fetched? -->  
+
+* [Prerequisites](#prerequisites)
+* [Step 1: Fetch all transactions during a specific period](#step-1-fetch-all-transactions-during-a-specific-period)
+* [Step 2: Fetch transactions of certain types during a specific period](#step-2-fetch-transactions-of-certain-types-during-a-specific-period)
+* [Related task](#related-task)
+* [Related API reference](#related-api-reference)
+
+## Prerequisites
+* Make sure that authorization is set up using [Authorization OAuth2 API](../../authorization.adoc). 
+<!-- to be continued if any -->
+
+## Step 1: Fetch all transactions during a specific period
+To fetch all transactions from a merchant's Zettle account during a specific period, you need to fetch them from their Zettle liquid account.
+<!-- Check with team Ledger: Is it correct that a PAYMENT transaction can have a PAYOUT state? When I fetched all transactions, I got transactions that include PAYOUT. How can the integrator tell which PAYOUT is from which transaction type? --> 
+
+1. Optional: Fetch your organisation UUID. 
+   > **Tip:** You don't need to fetch the organisation UUID, as you can specify the organisation UUID as `self` in requests that require it.
+
+    ```
+    GET users/me
+    ```
+   Example:
+       
+   The following example response returns organisation UUID `ab305d54-75b4-431b-adb2-eb6b9e546013`.
+
+    ```
+    {
+        "uuid": "de305d54-75b4-431b-adb2-eb6b9e546014",
+        "organizationUuid": "ab305d54-75b4-431b-adb2-eb6b9e546013"
+    }
+    ```
+
+2. Fetch all transactions from the merchant's Zettle liquid account.
+      > **Tip:** You specify the organisation UUID as `self`.
+     
+   ```
+   GET /organizations/{organizationUuid}/accounts/liquid/transactions?{start}&{end}
+   ```
+
+   Example:
+   
+   The following example request fetches all transactions from the merchant's Zettle liquid account from 1 January, 2020 to 7 June, 2021.
```

Comment:
As mentioned in previous comments  word "all" should be used carefully. In this case we are not fetching all transactions just a selection "from 1 January, 2020 to 7 June, 2021"